src/map/skill.cpp

```diff
@@ -17849,17 +17868,17 @@ int skill_attack_area(struct block_list *bl, va_list ap)
 int skill_clear_group(struct block_list *bl, int flag)
 {
 	struct unit_data *ud = NULL;
-	struct skill_unit_group *group[MAX_SKILLUNITGROUP];
-	int i, count = 0;
+	std::vector<std::shared_ptr<s_skill_unit_group>> group;
+	int count = 0;
```

Comment:
I would put this below ud check.
Think about removing the count (until the end of the function) and using push_back instead.